app/utils/ElasticClient.scala

```diff
@@ -36,7 +36,8 @@ object ElasticClient extends LazyLogging {
       s"elasticsearch://${elasticConfig.host}:${elasticConfig.port}?ssl=${elasticConfig.ssl}"
     ), new RequestConfigCallback {
       override def customizeRequestConfig(requestConfigBuilder: Builder) = {
-        requestConfigBuilder
+        // https://github.com/sksamuel/elastic4s/issues/1261
+        requestConfigBuilder.setConnectionRequestTimeout(10000)
```

Comment:
It would presumably be beneficial to make this hard-coded timeout value a configuration parameter.